src/zenml/logging/step_logging.py

```diff
@@ -81,6 +81,9 @@ def prepare_logs_uri(
     if log_key is None:
         log_key = str(uuid4())
 
+    if step_name is None:
+        step_name = "pipeline_runs"
```

Comment:
No? 😄 Maybe we change the attribute name to something that actually represents what it is?